Ginger/Ginger/Agents/AddAgentWizardLib/AddAgentDetailsPage.xaml.cs

```diff
@@ -53,8 +53,13 @@ public void WizardEvent(WizardEventArgs WizardEventArgs)
 
                     xPlatformTypeComboBox.SelectionChanged += xPlatformTypeComboBox_SelectionChanged;
                     App.FillComboFromEnumVal(xPlatformTypeComboBox, mWizard.Agent.Platform);
-                    xPlatformTypeComboBox.SelectedIndex = 0;
 
+                    //Removing ASCF type from combobox list
```

Comment:
Try to directly removing the specific value instead of setting and then removing value from 0th index
xPlatformTypeComboBox.Items.Remove(GingerCoreNET.SolutionRepositoryLib.RepositoryObjectsLib.PlatformsLib.ePlatformType.ASCF);

Reply:
I tried but it was not working for me. that's first i get the index of ASCF and remove it and before removing setting the index to 0.